frontend/src/libs/components/profile-settings-sidebar/profile-settings-button/styles.module.scss

```diff
@@ -0,0 +1,27 @@
+.profile-button {
+  display: flex;
+  align-items: center;
+  min-width: 317px;
+  padding: 3px 26px 3px 3px;
+
+  // Was not present in the variables file, got from figma
+  background-color: rgb(255 255 255 / 50%);
+  border: none;
+  border-radius: 20px;
+
+  // Was not present in the variables file, got from figma, the closest color is --white-alpha-80
+  box-shadow: 0 4px 23px #e5e1ff;
+  cursor: pointer;
+}
+
+.profile-button > img {
+  margin-right: 20px;
+}
+
+.profile-button-text {
+  color: var(--gray-300);
+  font-weight: 600;
+  font-size: 16px;
+  font-family: Montserrat, sans-serif;
```

Comment:
I think it's better to set basic font-family in body element in scaffolding to not set it every time in other places, but there's still no merged pull request with this approach

Reply:
We already have default font family in body element, so you can remove it)

Reply:
Resolve this comment